docs/Sprint-Review/sprint-33-summary.md

```diff
@@ -0,0 +1,48 @@
+# Sprint 33 Summary
+** 10/27/2021 - 11/09/2021 **
+
+## Sprint 33 Goals
+
+- ACF AMS
+- ZAP Scan
+- Resubmission Research Sessions
+- Django Admin - Figure out what a11y is left
+- Secret Key Mitigation Tickets
+
+## Closed/Merged/Completed (Done/Demo, Closed)
+- [#892](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/892) - As an OFA Admin, I want an accessible, 508-compliant user interface for managing permissions
+- [#536](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/536) - [Design] Sitemap
+- [#1013](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1013) - Break up Circle CI Jobs Using Labels (Hotfix)
+- [#1367](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1367) - Fine tune OWASP ZAP Scanner Usage
+- [#1271](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1271) - Resubmission Notification Flow
+
+
+## Submitted (QASP Review, OCIO Review)
+- [#1347](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1347) - SPIKE: Investigate using separate databases/buckets in shared services in the Cloud.gov dev space
+
+## Moving to Next Sprint (Raft Review, In Progress, Current Sprint Backlog)
+
+**Blocked**
+- n/a
+
+**Raft Review**
+- [#1108](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1108) - Fix state issues with Data Files Download button
+- [#1331](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1271) - Change splash page background image
+- [#888](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/888) - TDP Permissions Model
```

Comment:
```suggestion

```
I see the epic #888 is in the raft review column but I think it should actually be in product backlog.